README.md

```diff
@@ -5,11 +5,33 @@ See https://www.nanorfe.com for more info.
 Below is information for building the firmware on Linux.
 
 ## Installing the compiler
-On any recent Debian based installation:
+On any recent Ubuntu based installation:
 ``` 
 apt install arm-gcc-none-eabi python3-serial
 ```
 
+On Debian systems, you need to install the gnu ARM toolchain:
```

Comment:
I suggest making this a chapter on installing the latest arm gnu toolchain. @gabriel-tenma-white Is there a preferred toolchain currently?

Reply:
gcc-arm-none-eabi is available in debian: https://packages.debian.org/search?keywords=gcc-arm-none-eabi&searchon=names&suite=stable&section=all

Reply:
Yes, but I swapped the gcc and arm parts when creating the first README. So 
**arm-gcc-none-eabi** should be **gcc-arm-none-eabi**. 

So the command line in the current README does not work, letting @douardda come up with upstream gnu ARM toolchain. The Debian version is basically just an older version of the gnu ARM toolchain.

Reply:
So it the version on debian is ok or should the readme recommend using the upstream version? Any way, I'll rework my PR.

Reply:
I use the Debian version and it works for me, so that is fine. I think Ubuntu 20.04 ships GCC9.
That is why I asked @gabriel-tenma-white for any preference.

Reply:
@wutje ok so the rework of the PR I've done with proper instructions for debian based systems as well as a paragraph for the upstream arm toolchain version should be ok, right?

Reply:
Yes all is OK for me now. I will drop my PR since you fixed my typo as well.